CONTRIBUTING.md

```diff
@@ -394,7 +394,14 @@ Once the app has started you should be able to pull up these URLs in your web br
 * GraphQL Playground - http://localhost:5000/graphql
 * MailHog - http://localhost:8025/ - all dev environment emails are captured and viewable through this Mailhog interface 
 
-Note, MailHog is not started automatically in manual mode.  The easiest way to do that is via Docker: `docker run --rm --network host mailhog/mailhog`, but if you prefer to install it manually, instructions are on their [repository](https://github.com/mailhog/MailHog) 
+Note, MailHog is not started automatically in manual mode.  The easiest way to do that is via Docker: `docker run --rm --network host mailhog/mailhog`, but if you prefer to install it manually, instructions are on their [repository](https://github.com/mailhog/MailHog)
+</details>
+
+<details><summary><b>Step 5</b> - Log In As Different Users</summary>
```

Comment:
```suggestion
<details><summary><b>Step 5</b> - Log In As Different Users</summary>

```
Somehow lack of this whitespace makes backticks not work as intended when markdown is rendered.

Reply:
Huh, that takes me back.  I ran into this exact issue when I was trying to convert the old weird custom fcc markdown into the more spec compliant md we use these days.  Fun times.

Anyway, thanks for catching it!